locales/zh-CN/editor.properties

```diff
@@ -125,7 +125,7 @@ CMD_TOGGLE_QUICK_DOCS=快速文档
 
 # Drag and Drop
 
-DND_MAX_FILE_SIZE_EXCEEDED=文件超过支持的最大大小: 3MB
+DND_MAX_FILE_SIZE_EXCEEDED=文件超过支持的最大大小：3MB
```

Comment:
it looks like you accidentally changed this file. Can you revert it so it doesn't show up as part of this commit?